assets/js/application.js

```diff
@@ -10,8 +10,8 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
-//= require jquery-1.7.1.min
-//= require jquery-ui-1.8.18.custom.min
+//= require jquery-3.7.1.min
+//= require jquery-ui-1.14.1.min
```

Comment:
The first line of the commit message talks about adding two files. That's not the only change this commit makes, though, is it? It upgrades _at the same time_.

Let's just stop vendoring in the jQuery `.js` files. There are perfectly fine CDN versions of them; As a bonus, browsers can rely on the same `.js` files across sites, they don't need to cache a separate copy _just_ for git-scm.com.